src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -28,6 +28,7 @@ public partial class ConfigurationControl : IssueConfigurationControl
         public ConfigurationControl()
         {
             InitializeComponent();
+            DataContext = vmConfig;
```

Comment:
>vmConfig [](start = 26, length = 8)

Where is vmConfig come from?

Reply:
in the same class, at this point it equals 
`public ConfigurationViewModel vmConfig { get; private set; } = new ConfigurationViewModel();`

once you switch to connection tab, this code is called
  ```
private async void ChangeStates(ControlState state){
            vmConfig.Avatar = AzureDevOps.Avatar;
            vmConfig.DisplayName = AzureDevOps.DisplayName;
            vmConfig.Email = AzureDevOps.Email;
            vmConfig.State = state;
         ........
```